index.js

```diff
@@ -195,6 +196,10 @@ Paper.prototype.cdnify = function (path) {
         path = path.substr(8, path.length);
     }
 
+    if (sessionId) {
+        return [cdnUrl, 'stencil', versionId, configId, sessionId, path].join('/');
```

Comment:
🍹 `Path.join(cdnUrl, 'stencil', versionId, configId, sessionId, path)`? Not picky though.